### PR TITLE
Prepare tracing-appender for release

### PR DESCRIPTION
**Pull request by zekisherif**
Following the release process for https://github.com/tokio-rs/tracing/blob/master/CONTRIBUTING.md and these changes are the only things left to commit before release.

I'd like to hopefully get this released today so we can start using this crate internally.

Note:
I need to get PR #703 and PR #678 merged before release.

### CODE REVIEW

**Comment by hawkw** [comment]
I think there is a little more to do to get this ready to release. Let's also update the `Cargo.toml` to contain the standard set of metadata that other crates in the project have:
https://github.com/tokio-rs/tracing/blob/5a971befacc233612ec6577c9bae6188c79926ad/tracing/Cargo.toml#L13-L27

Also, I'd like to give this crate a quick final API pass before publishing in case we need to change anything up.

> I'd like to hopefully get this released today so we can start using this crate internally.

If you want to use the crate before it's ready to release, are you able to take a git dependency temporarily?

**Comment by hawkw** [comment]
> Following the release process for https://github.com/tokio-rs/tracing/blob/master/CONTRIBUTING.md and these changes are the only things left to commit before release.
> 
> I'd like to hopefully get this released today so we can start using this crate internally.
> 
> Note:
> I need to get PR #703 and PR #678 merged before release.

I don't think it's necessary to block a crates.io release on the benchmarks. They aren't part of the crate's API and will primarily be useful internally for development purposes. I would like to merge the tests first, though.

**Comment by zekisherif** [comment]
> > I'd like to hopefully get this released today so we can start using this crate internally.
> 
> If you want to use the crate before it's ready to release, are you able to take a git dependency temporarily?

Unfortunantely we aren't able to take a git dependency. The only way for us to use this is if there is a crate available for us.

File: tracing-appender/Cargo.toml [hunk 1]
```diff
@@ -2,12 +2,20 @@
 name = "tracing-appender"
 version = "0.1.0"
 authors = ["Zeki Sherif <zekshi@amazon.com>"]
```
Comment (hawkw) [inline tracing-appender/Cargo.toml:4]: Can you add `"Tokio Contributors <team@tokio.rs>"` here as well, please? Thanks!
```suggestion
authors = [
    "Zeki Sherif <zekshi@amazon.com>",
    "Tokio Contributors <team@tokio.rs>"
]
```

File: tracing-appender/Cargo.toml [hunk 2]
```diff
@@ -2,12 +2,20 @@
...
 description = """
 Provides an `appender` that writes to a file
 """
+categories = [
+    "development-tools::debugging",
+    "development-tools::profiling",
+    "asynchronous",
+]
+keywords = ["logging", "tracing", "file-appender", "non-blocking-writer"]
+edition = "2018"
```
Comment (hawkw) [inline tracing-appender/Cargo.toml:18]: :thumbsup:

File: tracing-appender/Cargo.toml [hunk 3]
```diff
@@ -2,12 +2,20 @@
...
+license = "MIT"
+readme = "README.md"
 repository = "https://github.com/tokio-rs/tracing"
 homepage = "https://tokio.rs"
 description = """
 Provides an `appender` that writes to a file
 """
+categories = [
+    "development-tools::debugging",
+    "development-tools::profiling",
```
Comment (hawkw) [inline tracing-appender/Cargo.toml:14]: I'm not totally sure if this needs the "profiling" tag, but :woman_shrugging:

File: tracing-appender/Cargo.toml [hunk 4]
```diff
@@ -2,12 +2,20 @@
 name = "tracing-appender"
 version = "0.1.0"
 authors = ["Zeki Sherif <zekshi@amazon.com>"]
-edition = "2018"
+license = "MIT"
+readme = "README.md"
 repository = "https://github.com/tokio-rs/tracing"
 homepage = "https://tokio.rs"
 description = """
 Provides an `appender` that writes to a file
```
Comment (hawkw) [inline tracing-appender/Cargo.toml:10]: I don't think this description is entirely accurate. This crate provides a file appender _and_ utilities for making a writer non-blocking. Can we update the description to something a little more general?

Comment (zekisherif) [reply tracing-appender/Cargo.toml]: updated

**Comment by zekisherif** [comment]
Awesome! Thanks for all your help in the countless reviews.

I'll have a follow-up PR out later today.

**Pull request merged**